.buildkite/pipeline.yml

```diff
@@ -348,8 +348,8 @@ steps:
       # GPU RUNS: slabplanet
       - label: "GPU Slabplanet: albedo from function"
         key: "gpu_slabplanet_albedo_function"
-        command: "julia --color=yes --project=experiments/AMIP/ experiments/AMIP/coupler_driver.jl --config_file $CONFIG_PATH/slabplanet_albedo_function.yml"
-        artifact_paths: "experiments/AMIP/output/gpu/slabplanet/slabplanet_albedo_function_artifacts/*"
+        command: "julia --color=yes --project=experiments/AMIP/ experiments/AMIP/coupler_driver.jl --config_file $CONFIG_PATH/gpu_slabplanet_albedo_function.yml"
```

Comment:
do you prefer to have a separate YAML file for the GPU run? We could just use the same config file for both CPU and GPU runs

Reply:
Yes, otherwise this sometimes leads to race conditions when the CPU and GPU runs are sharing the same output folders in the same build (e.g. https://buildkite.com/clima/climacoupler-ci/builds/2550).

Reply:
got it! good to know

Reply:
I agree though, it would be nicer to have a way of calling the same run yml with different architectures in our CI. Maybe we can bring this up in the next software meeting, since the other repos use the same system as us.